src/style.css

```diff
@@ -1,3 +1,63 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto:wght@500&display=swap");
+
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your
web document